ncm-syslog/src/test/perl/configure.t

```diff
@@ -43,6 +43,7 @@ random stuff
 *.* other action
 user.crit\tawesome
 mail.debug\tawesome
+uucp,news.crit\t/var/log/spooler
```

Comment:
need to add this line either to orig in the next tests, or always to the expected value